hal/halx86/smp/amd64/halmacpi.rc

```diff
@@ -0,0 +1,5 @@
+#define REACTOS_VERSION_DLL
+#define REACTOS_STR_FILE_DESCRIPTION  "X64 Multiprocessor Hardware Abstraction Layer"
+#define REACTOS_STR_INTERNAL_NAME     "halmacpi"
```

Comment:
Do we actually need `smp` directory, in addition to the `mp` one?

Reply:
The mp folder isn't going to be there forever, I was advised just to keep it for now. The code inside the MP folder is all meant for the older MP tables and I just decided to keep unchanged with the exception of me moving the APIC header file that was meant for them into that folder. I'm going to be focusing my time on halmacpi for now instead of halmp.